src/app/settings/settings.component.html

```diff
@@ -56,9 +56,14 @@
       <br />
       <select name="theme" id="theme" (change)="selectChangeHandler($event)" class="form-control selector">
         <option *ngIf="whichTheme == 'dark'" value="dark">Dark Theme</option>
+        <option *ngIf="whichTheme == 'dark'" value="iceydark">Icey Dark Theme</option>
         <option *ngIf="whichTheme == 'light'" value="light">Light Theme</option>
         <option *ngIf="whichTheme == 'light'" value="dark">Dark Theme</option>
+        <option *ngIf="whichTheme == 'light'" value="iceydark">Icey Dark Theme</option>
         <option *ngIf="whichTheme == 'dark'" value="light">Light Theme</option>
+        <option *ngIf="whichTheme == 'iceydark'" value="iceydark">Icey Dark Theme</option>
```

Comment:
Along the lines of my other comment, I think all of these `<option>` can be simplified by removing the `*ngIf`:
```ts
<option value="dark">Dark Theme</option>
<option value="light">Light Theme</option>
<option value="iceydark">Icey Dark Theme</option>
```

Reply:
Without the ngIf, it will default to "Dark Theme" at the top and show as selected even if the user has the Light Theme on for example, I am sure there is possibly a better method to do this, but currently it will show the selected as the users currently selected theme.

Reply:
Oh I see what you mean. Typically this is fixed by setting the value of the `<select>` to whatever the current theme is, so the selected one always appears. I think this should probably work:

```ts
<select [value]="whichTheme" ... >
```

(keeping all the other attributes, just adding the `[value]="whichTheme"` part to it)